i18n-helpers/src/lib.rs

```diff
@@ -311,7 +334,7 @@ pub fn group_events<'a>(events: &'a [(usize, Event<'a>)]) -> Vec<Group<'a>> {
                 }
             }
 
-            Event::Html(s) => {
+            Event::Html(s) | Event::InlineHtml(s) => {
```

Comment:
Nice, this is the new kind of HTML you mentioned to me?

I think this is the right thing to do right now, but I wonder if we can make good use of this to deal with #125?

Reply:
Yes, that what I was thinking about as well. Can be done once this PR is merged.

Reply:
That sounds great!